.github/scripts/createAndPostTag.js

```diff
@@ -73,12 +69,19 @@ async function createAndPostTag(params) {
     await createTag(github, owner, repo, newVersion, releaseBranchSha);
 
     // Update the RELEASE_VERSION repo variable
-    await github.rest.actions.updateRepoVariable({
-      owner,
-      repo,
-      name: "RELEASE_VERSION",
-      value: newVersion,
-    });
+	await github.rest.actions.updateRepoVariable({
+	  owner,
+	  repo,
+	  name: "RELEASE_VERSION",
+	  value: newVersion,
+	});
+
+    // Output previous version to the GitHub actions workflow context
+    // This will be used by the workflow that sets up the release notes
+    core.setOutput("previousVersion", previousVersion);
+
+	// this output will be the tag used for the staging deploy
```

Comment:
this comment should not be indented.

Reply:
I'm updating these files with prettier now